src/lib/header-bidding/utils.ts

```diff
@@ -84,6 +87,9 @@ export const containsLeaderboard = (sizes: HeaderBiddingSize[]): boolean =>
 export const containsBillboard = (sizes: HeaderBiddingSize[]): boolean =>
 	contains(sizes, createAdSize(970, 250));
 
+export const containsSkyscraper = (sizes: HeaderBiddingSize[]): boolean =>
```

Comment:
Is this function used anywhere? Does it need to be used in `getKargoPlacementId`?

Reply:
Nice one, thanks!